fs/walk.ts

```diff
@@ -97,7 +97,7 @@ export type { WalkEntry };
  * Recursively walks through a directory and yields information about each file
  * and directory encountered.
  *
- * The file paths are absolute paths. The root directory is included in the
+ * The file paths are relative paths. The root directory is included in the
```

Comment:
Looks like it depends on the input. If you pass an absolute path to `walk`, then it seems returning absolute paths.

```ts
import { walk } from "@std/fs";
for await (const item of walk("/Users/kt3k/denoland/std/regexp")) {
  console.log(item);
}
```

```
{
  path: "/Users/kt3k/denoland/std/regexp",
  name: "regexp",
  isFile: false,
  isDirectory: true,
  isSymlink: false
}
{
  path: "/Users/kt3k/denoland/std/regexp/escape_test.ts",
  name: "escape_test.ts",
  isFile: true,
  isDirectory: false,
  isSymlink: false
}
...
```

Reply:
You are right. I changed the expression as following:
> The root path determines whether the file paths is relative or absolute.